fuzztests/checks.go

```diff
@@ -305,9 +308,15 @@ func CheckConvexHull(t *testing.T, want UnaryResult, g geom.Geometry) {
 	t.Run("CheckConvexHull", func(t *testing.T) {
 		got := g.ConvexHull()
 		want := want.ConvexHull
+
+		// PostGIS retains 3D and M coordinates for convex hull, which is
```

Comment:
I think there needs to be a common parsed symbol to segregate PostGIS-related code. Although having notes in the comments is good, I think `if ComplyStrictlyWithPostGIS` (or something similar) would be a better way to separate and contain all code intended to deal with those differences (And a common name for such a flag would make it searchable).

Reply:
In this case, the behaviour here is more to work around the differences so that we don't get a false negative test failure. I'm less keen to add "compatibility mode" flags (I think that's what you're referring to?) since it adds a bit of complexity, and I'm not sure right now what the usecase for differing compatibility would be (postgis vs libgeos vs potentially others).